build.gradle

```diff
@@ -1538,7 +1540,7 @@ project(':forge') {
         }
     }
     // Since we need the modules in the bootstrap, we need to make sure they are compiled before we do each run
-    afterEvaluate { prepareRuns.dependsOn(':fmlcore:jar', ':fmlloader:jar', ':javafmllanguage:jar', ':mclanguage:jar') }
+    afterEvaluate { prepareRuns.dependsOn(':fmlcore:jar', ':fmlloader:jar', ':javafmllanguage:jar', ':lowcodelanguage:jar', ':mclanguage:jar') }
```

Comment:
It looks to me like this is a duplication of PACKED_DEPS, so it should just reference PACKED_DEPS directly.

Reply:
This is how it is in javafmllanguage. Fix it there.